python-checks/src/main/java/org/sonar/python/checks/MethodShouldBeStaticCheck.java

```diff
@@ -53,12 +54,33 @@ public void init() {
   public void visitNode(AstNode node) {
     if (CheckUtils.isMethodDefinition(node) && !alreadyStaticMethod(node) && hasValuableCode(node)){
       String self = getFirstArgument(node);
-      if (self != null && !isUsed(node, self)){
+      if (self != null && !isUsed(node, self) &&!isDocStringsAndNotImplementedError(node)){
         getContext().createLineViolation(this, MESSAGE, node.getFirstChild(PythonGrammar.FUNCNAME));
       }
     }
   }
 
+  private static boolean isDocStringsAndNotImplementedError(AstNode funcDef) {
```

Comment:
@vilchik-elena It was for consistency with the naming of the other methods in the class